src/components/draft/Explorer.js

```diff
@@ -45,16 +48,15 @@ const componentCss = css`
   cursor: pointer;
   user-select: none;
   padding: 2px;
-  margin-left: 20px;
   color: var(--color-text-primary);
-  border-radius: 2px;
+  border-radius: 3px;
 
   &[data-selected='true'] {
     font-weight: bold;
   }
 
   &:hover {
-    /* background-color: var(--color-background-highlight); */
+    background-color: #dddddf;
```

Comment:
Why not put this in its own css var?

Reply:
At some point I want to revisit how we're doing the vars.